examples/example.md

```diff
@@ -1,136 +1,160 @@
 
 ```bash
 kind create cluster --name crossplane-example
 kubectl config use-context kind-crossplane-example
 ```
 
-**2. Create namespace for the crossplane ecosystem:**
+### Create namespace for the crossplane ecosystem
+
+To create namespace for the crossplane ecosystem, run the following command:
 
 ```bash
 kubectl create namespace crossplane-system
 ```
 
-**3. Install crossplane via helm:**
+### Install Crossplane via helm
+
+To create a cluster in case of using ``helm``, run the following command:
 
 ```bash
 helm repo add crossplane-stable https://charts.crossplane.io/stable
 helm repo update
 helm install crossplane --namespace crossplane-system crossplane-stable/crossplane
 ```
 
-**4. Register CRDs into k8s cluster:**
+### Register CRDs into k8s cluster
+
+To register CRDs into k8s cluster, run the following command:
 
 ```bash
 kubectl apply -f package/crds/ -R
 ```
 
-> Note: Before continuing, you can check if `kubectl get providers` will recognize the CRDs of type `providers`. The command should return `No resources found`.
+{% hint style="info" %}
+**Note:** Before continuing, you can check if ``kubectl get providers`` will recognize the CRDs of type ``providers``. The command should return: ``No resources found``.
+{% endhint %}
 
-**5. Install ProviderConfig, for credentials:**
+### Install ProviderConfig for credentials
 
-```bash
-export BASE64_PW=$(echo -n "${IONOS_PASSWORD}" | base64)
-kubectl create secret generic --namespace crossplane-system example-provider-secret --from-literal=credentials="{\"user\":\"${IONOS_USERNAME}\",\"password\":\"${BASE64_PW}\"}"
-kubectl apply -f examples/provider/config.yaml
-```
+To install ``ProviderConfig`` for credentials, run the following commands:
 
-or
+  ```bash
+  export BASE64_PW=$(echo -n "${IONOS_PASSWORD}" | base64)
+  kubectl create secret generic --namespace crossplane-system example-provider-secret --from-literal=credentials="{\"user\":\"${IONOS_USERNAME}\",\"password\":\"${BASE64_PW}\"}"
+  kubectl apply -f examples/provider/config.yaml
+  ```
 
-```bash
-kubectl create secret generic --namespace crossplane-system example-provider-secret --from-literal=credentials="{\"token\":\"${IONOS_TOKEN}\"}"
-kubectl apply -f examples/provider/config.yaml
-```
+OR
+
+  ```bash
+  kubectl create secret generic --namespace crossplane-system example-provider-secret --from-literal=credentials="{\"token\":\"${IONOS_TOKEN}\"}"
+  kubectl apply -f examples/provider/config.yaml
+  ```
 
-_Note_: You can overwrite the default IONOS Cloud API endpoint, by setting `host_url` option in credentials
-struct: `--from-literal=credentials="{\"host_url\":\"${IONOS_API_URL}\"}"`
+{% hint style="info" %}
+**Note:**   You can overwrite the default IONOS Cloud API endpoint, by setting ``host_url`` to: ``--from-literal=credentials="{\"host_url\":\"${IONOS_API_URL}\"}"``.
+{% endhint %}
 
-**6. Install Crossplane Provider IONOS Cloud**
+### Install Crossplane Provider IONOS Cloud
```

Comment:
Should this be _Install Crossplane Provider **for** IONOS Cloud_?

Reply:
Done, thanks a lot